Repositories/CourierRepository.cs

```diff
@@ -1,32 +1,34 @@
-﻿using shipping_service.Persistence.Entities;
+﻿using shipping_service.Persistence.Database;
+using shipping_service.Persistence.Entities;
 
 namespace shipping_service.Repositories
 {
     public class CourierRepository : ICourierRepository
     {
-        public async Task<List<Courier>> GetAsync()
-        {
-            return new List<Courier>();
-        }
+        private readonly DatabaseContext context;
 
-        public async Task<Courier> GetAsync(ulong id)
+        public CourierRepository(DatabaseContext ctx)
         {
-            return new Courier();
+            context = ctx;
         }
 
-        public async Task<Courier> CreateAsync(Courier courier)
+        public IQueryable<Courier> Couriers => context.Couriers;
+
+        public async Task CreateAsync(Courier courier)
         {
-            return new Courier();
+            await context.AddAsync(courier);
+            await context.SaveChangesAsync();
         }
 
-        public async Task<Courier> UpdateAsync(ulong id, Courier courierIn)
+        public async Task UpdateAsync(Courier courier)
         {
-            return new Courier();
+            await context.SaveChangesAsync();
         }
 
-        public async Task<ulong> DeleteAsync(ulong id)
+        public void DeleteAsync(Courier courier)
```

Comment:
This should be either be made async, or remove Async in the name for less confusion.